src/app/backend/deploy.go

```diff
@@ -86,13 +96,23 @@ func DeployApp(spec *AppDeploymentSpec, client *client.Client) error {
 		Labels:      labels,
 	}
 
+	containerSpec := api.Container{
+		Name:  spec.Name,
+		Image: spec.ContainerImage,
+	}
+
+	if spec.ContainerCommand != nil {
+		containerSpec.Command = []string{*spec.ContainerCommand}
+	}
+
+	if spec.ContainerCommandArgs != nil {
+		containerSpec.Args = []string{*spec.ContainerCommandArgs}
```

Comment:
I know. I've been thinking about this. The reason I decided to go with just a string for our API is that this is presentation layer API, so it should be as close to the presentation layer as possible. It is implementation detail, that the actual backend uses array of commands. The thing that matters is that the UI has just input for one command.
When we change the UI to allow for multiple commands, then updating the presentation layer is OK.